app/src/main/java/org/oppia/android/app/walkthrough/topiclist/WalkthroughTopicListFragmentPresenter.kt

```diff
@@ -8,6 +8,8 @@ import android.view.ViewGroup
 import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.walkthrough_activity.*
```

Comment:
Try to avoid such imports, we can access id's using the binding obj we have.

Reply:
Sure, will definitely fix it

Reply:
I think we need this for ```addOnScrollListener(object : RecyclerView.OnScrollListener())```

Reply:
Try to do it with the DataBinding. 
As an example in this file we have `android:progress="@{viewModel.currentProgress}"`